build/make/coder.mk

```diff
@@ -28,10 +28,9 @@ CHANGELOG_FILE=${WORKDIR}/CHANGELOG.md
 TEMPLATE_RELEASE_TAR_GZ=${RELEASE_DIR}/${TEMPLATE_NAME}-template.tar.gz
 
 TEST_WORKSPACE_PREFIX?=test-${TEMPLATE_NAME}
-# only set if coder is installed, so that there is no problem with build and release targets
-CODER_USER?=$(shell if [ -x "$(command -v coder)" ]; then coder users show me -o json | jq -r '.username'; else echo ""; fi )
+CODER_USER?=$(shell . ${CODER_LIB_PATH} && getCoderUser)
 
-CONTAINER_BIN?=$(shell if [ -x "$(command -v podman)" ]; then echo "podman"; else echo "docker"; fi)
+CONTAINER_BIN?=$(shell . ${CODER_LIB_PATH} && getContainerBin)
```

Comment:
This is an unusual way (it works though, so I am okay with that :smile:). My guess is to circumvent the zsh/bash execution problem with execution guards?

I thought we had a solution for that?

If you use the exec guard way I would also consider using `getopts` like in [this article](https://www.redhat.com/sysadmin/arguments-options-bash-scripts) to simplify script usage.